puzzles/models.py

```diff
@@ -138,16 +135,6 @@ def update_metadata(self, new_name, new_url, new_is_meta):
 
         self.save()
 
-        def update_sheets():
-            if is_new_url:
-                google_api_client = GoogleApiClient.getInstance()
-                if google_api_client:
-                    google_api_client.add_puzzle_link_to_sheet(self.url, self.sheet)
-            for meta in self.metas.all():
-                GoogleApiClient.update_meta_sheet_feeders(meta)
-
-        transaction.on_commit(update_sheets)
-
```

Comment:
Why deleting this?

Reply:
google_api_lib now directly updates Puzzles, so I moved all calls to google_api_lib to api/views.py to avoid circular dependencies. This part was moved to https://github.com/cardinalitypuzzles/smallboard/blob/2fe8a5d0e11f39156e56a51d30c8a49f7770e5d4/api/views.py#L160-167